### PR TITLE
adds patch files and steps for THREESCALE-3847

### PR DESCRIPTION
**Pull request by kevprice83**
@hallelujah I have made a few changes to the steps you shared in the JIRA, please let me know if this looks good. I have only tested against CDB so far and can add non-CDB in the same PR later this week.

### CODE REVIEW

**Comment by hallelujah** [comment]
@kevprice83 Thank you!
I think we must open another issue than referring to the closed THREESCALE-3847.
Specifying exactly what is the issue because comments are not ideal for context.

I'd specify there:
- topic: can't work with service name
- version impacted
- decision which one we should fix (this should not be us to decide but product)

File: database.rb [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+      end
+
+      protected
+
+      def username
+        @config['username']
+      end
+    end)
+  end
+end
```
Comment (hallelujah) [inline database.rb:210]: I am not sure this file should be committed, at least not in the root path

**Comment by eguzki** [comment]
we are about to tag 2.9 CR1, JFYI.  @kevprice83 @hallelujah what should be done with this PR? It is ok if you say, `leave it open`

**Comment by kevprice83** [comment]
@eguzki @hallelujah I don't know if we want to use this PR (tidied up) or open a new PR but the objective is to provide some steps that work for 2.7 & 2.8 when the service name parameters do not match as detailed in the README. I think nothing effects this PR in the next release and if required we will add this as a patch also. I will review the PR today for tidying up based on your previous comments @hallelujah

**Comment by mayorova** [comment]
The more relevant JIRA for this PR is actually https://issues.redhat.com/browse/THREESCALE-7315

Also, I tried the updated SQL queries that are provided in the patch and they did not work for the described issue (I guess they worked for a similar scenario, but for sure it's not universal).

In addition to that, since 2.12 the SQL for connection probe were moved to [lib/system/connection_probe.rb](https://github.com/3scale/porta/blob/3scale-2.12.0-GA/lib/system/connection_probe.rb#L29-L34), so the patch would not work for it either.

I'd suggest to close this PR, continue to assist customers who may be affected by this through the KCS https://access.redhat.com/solutions/4966881 and work on the proper fix for the issue https://issues.redhat.com/browse/THREESCALE-7315.

**Comment by eguzki** [comment]
And templates have been dropped in 3scale 2.13

**Pull request closed without merging**